src/main/resources/templates/cypher/list.html

```diff
@@ -36,16 +35,17 @@ <h2>Šifry</h2>
                 <td th:text="${cypher.codeword}">codove slovo</td>
                 <td th:text="${cypher.trapCodeword}">Past</td>
                 <td th:text="${cypher.bonusContent}">Bonus</td>
-                <td>
-                    <a href="#" th:href="@{${cypher.mapAddress}}" target="_blank" class="btn btn-secondary">Mapa</a>
-                </td>
                 <td>
                     <button class="btn btn-primary dropdown-toggle" type="button" data-toggle="dropdown">Akce
                         <span class="caret"></span>
                     </button>
                     <ul class="dropdown-menu">
                         <li><a href="#" th:href="@{/admin/hint(cypherId=${cypher.cypherId})}" class="btn">Detail</a></li>
                         <li><a href="#" th:href="@{/admin/cypher/update/{cypherId}(cypherId=${cypher.cypherId})}" class="btn">Upravit</a></li>
+                        <li>
+                            <a th:if="${cypher.mapAddress != '' && cypher.mapAddress != null}" href="#" th:href="@{${cypher.mapAddress}}" target="_blank" class="btn">Mapa</a>
```

Comment:
Nestačil by jen ten <a blok? Že pokud mapa ne ude nastavena, tlačítko mapa nebude vidět?

Reply:
Šlo. Mám připravené obě varianty. Takže asi záleží na vás :)
Mě to přišlo zrovna pěknější tam to tlačítko mít jakoby neaktivní.

Reply:
Me je to asi jedno ;)